PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionJobUploadStorageCtrl.kt

```diff
@@ -242,14 +243,15 @@ class SparkSubmissionJobUploadStorageCtrl(val view: SparkSubmissionJobUploadStor
             }
     }
 
-    override fun getAzureBlobStoragePath(fullStorageBlobName: String?, container: String?): String? {
-        return if (StringUtils.isBlank(fullStorageBlobName) || StringUtils.isBlank(container)) null else
-            "wasbs://$container@$fullStorageBlobName/SparkSubmission/"
+    override fun getAzureBlobStoragePath(fullStorageBlobName: String?, container: String?, schema: String?): String? {
+        return if (StringUtils.isBlank(fullStorageBlobName) || StringUtils.isBlank(container) || StringUtils.isBlank(schema)) null
```

Comment:
I suppose here should send out `InvalidArgumentsException`, not just return `null`, which is hard to debug.